src/v3/src/transformer/button/transformIDPButtons.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { IDX_STEP } from '../../constants';
+import {
+  ButtonElement,
+  ButtonType,
+  DividerElement,
+  IWidgetContext,
+  TransformStepFnWithOptions,
+} from '../../types';
+import { loc } from '../../util';
+
+const PIV_TYPE = 'X509';
+
+// TODO: Implement CUSTOM IDP Buttons here
```

Comment:
The idea here is to group all IDP buttons in this transformer, the reason for this is because this is also how v2 groups them, PIV/CAC is just always placed as the first (when custom IDPs also exist). But since we have not implemented custom IDP buttons yet, I am marking this with a TODO, will create a ticket for custom IDP implementation and add here.

Reply:
I think we want some better unit test coverage of this button transformer or perhaps just the onClick function of the Button if possible